course3/homework-list/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+
+cmake_minimum_required (VERSION 3.5 )
+
+set(CMAKE_CXX_FLAGS "-std=c++14 -Wall")
+
+
+find_package( Boost 1.6 COMPONENTS program_options REQUIRED)
```

Comment:
I was using 'program_options' to try and see if I could get boost to work, answer: Linux - easy, Windows - not yet, still fighting it.